### ISSUE TITLE
lambdaj 2.3.3 is missing on maven central

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dschulten/lambdaj#73]
```
searching

http://search.maven.org/#search|ga|1|lambdaj

results only to

com.googlecode.lambdaj  lambdaj 2.3.1  all (6)  24-Aug-2010 pom  jar  javadoc.jar 
 sources.jar  


```

Original issue reported on code.google.com by `j.sare...@gmail.com` on 31 Jul 2011 at 10:52

**Comment by GoogleCodeExporter** [comment]
```
Unfortunately adding the lambdaj repo is not an option in my case, I need it on 
maven central. What is the issue you are facing when uploading to central?
```

Original comment by `dietrich...@gmail.com` on 15 Aug 2011 at 12:27

**Comment by GoogleCodeExporter** [comment]
```
Hello, are there any updates on this? I'd like to use also the latest version 
from maven central.
thanks
```

Original comment by `david.ga...@gmail.com` on 27 Sep 2011 at 12:50

**Comment by GoogleCodeExporter** [comment]
```
This is solved, 2.3.3 is in central now
```

Original comment by `dietrich...@gmail.com` on 22 Nov 2011 at 12:09

**Comment by GoogleCodeExporter** [comment]
Original comment by `mario.fu...@gmail.com` on 22 Nov 2011 at 1:38
- Changed state: **Fixed**
